src/main/java/carpet/script/api/Inventories.java

```diff
@@ -53,7 +53,14 @@ public static void apply(Expression expression)
         expression.addLazyFunction("item_category", 1, (c, t, lv) ->
         {
             ItemStackArgument item = NBTSerializableValue.parseItem(lv.get(0).evalValue(c).getString());
-            Value res = new StringValue(item.getItem().getGroup().getName());
+            Value res;
+
+            if(item.getItem().getGroup().getName()==null)
```

Comment:
well, its not the name that can be null, but the group, right?
should be 
```java
ItemGroup ig = item.getItem().getGroup();
Value res = (ig==null)?Value.NULL:new StringValue(ig.getName());
return (_c, _t) -> res;
```
right?

Reply:
But if group is null, wont getName() return null, or NullPointerException? Anyways that is faster.